base/shell/explorer/startmnusite.cpp

```diff
@@ -112,6 +115,56 @@ class CStartMenuSite :
         return FALSE;
     }
 
+    void FillDocumentsMenu(HMENU hDocMenu)
```

Comment:
this thing is a hack. there is a more proper way to do that

Reply:
OK.